src/schema.js

```diff
@@ -29,18 +29,18 @@ import {
 const Query = new GraphQLObjectType({
   name: 'BlogSchema',
   description: 'Root of the Blog Schema',
-  fields: () => ({
+  fields: {
```

Comment:
Here we use an arrow function for the lazy evaluation. I think we need to have it.  Could you do it for other places as well. (only when defining fields)